crates/conformance-test-utils/src/hive/genesis.rs

```diff
@@ -23,6 +37,120 @@ impl HiveGenesisConfig {
     }
 }
 
+/// Convert Hive Genesis Config to Madara Genesis Config
+///
+/// This function will:
+/// 1. Load the Madara genesis file
+/// 2. Construct a Starknet test sequencer
+/// 3. Declare Kakarot contracts
+/// 4. Add Kakarot contracts to Loader
+/// 5. Add Hive accounts to Loader
+/// 6. Serialize Loader to Madara genesis file
+pub async fn serialize_hive_to_madara_genesis_config(hive_genesis: HiveGenesisConfig) -> Result<(), IoError> {
+    // Load the Madara genesis file
+    let mut loader: GenesisLoader =
+        serde_json::from_str(&read_file_to_string("crates/conformance-test-utils/src/madara/genesis.json"))
+            .expect("Failed to load Madara genesis file");
+
+    // Get compiled path of contracts on loader
+    let mut compiled_path = PathBuf::from("");
+    if let ContractClass::Path { path, .. } = &loader.contract_classes[0].1 {
+        compiled_path = PathBuf::from(path);
+
+        // Remove filename
+        compiled_path.pop();
```

Comment:
We need to save the compiled contracts on this path in Madara, as indicated in the resulting `madara_genesis.json`

Reply:
You're right, indeed we'll have a problem if the first element would be a `ContractClass` variant, should we add this compiled path of loader as a parameter instead?